forms-flow-web/src/components/Form/Item/Edit.js

```diff
@@ -211,9 +221,7 @@ const Edit = React.memo(() => {
         );
       }
   };
-  // save form data to submit
-  const saveFormData = () => {
-    setFormSubmitted(true);
+  const setFormDataForSendingBackend = (form)=>{
```

Comment:
use better naming with comments for better readability and code quality, recommend something specific